TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/TagHelpers/UseEmptyFallbackTagHelper.cs

```diff
@@ -0,0 +1,21 @@
+using System.Text.Encodings.Web;
+using Microsoft.AspNetCore.Mvc.TagHelpers;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+
+namespace TeachingRecordSystem.SupportUi.TagHelpers;
+
+[HtmlTargetElement("span", Attributes = "use-empty-fallback")]
```

Comment:
```suggestion
[HtmlTargetElement("*", Attributes = "use-empty-fallback")]
```

Reply:
We could avoid adding extra `span`s if we can use this everywhere (thought it probably won't work on the `govuk-` elements unless we lower the `Order`)

Reply:
it didn't work when I tried with govuk-summary-list-row-value as it seemed to ignore any changes made to content when it came to rendering